website/templates/project/project_header.mako

```diff
@@ -3,10 +3,22 @@
 %>
 
 <div id="projectBanner" >
-    <header class="subhead" id="overview">
-        <nav id="projectSubnav" class="navbar osf-project-navbar" role="navigation">
-            <div class="container">
 
+
+     % if node['category_short']  == 'share window':
```

Comment:
There doesn't appear to be anything setting a node's category to `'share window'`